src/modules/jobs.rs

```diff
@@ -15,12 +15,24 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         .trim()
         .parse::<i64>()
         .ok()?;
-    if num_of_jobs == 0 {
+
+    if config.threshold < 0 {
+        log::warn!(
+            "threshold in [jobs] ({}) was less than zero",
+            config.threshold
+        );
+        return None;
+    }
+
+    if num_of_jobs == 0 && config.threshold > 0 {
```

Comment:
```suggestion
    if num_of_jobs < config.threshold {
```

Reply:
The previous behavior was that if there are _any_ background jobs running, then the module would show a symbol, regardless of the threshold. Should we try to keep this same behavior?

Reply:
Oh right, good catch! :+1: Let's keep existing behavior.